packages/relay-compiler/README.md

```diff
@@ -68,7 +68,7 @@ file sources, and "listen" to the file changes in the "watch" mode. If
   enabling this the babel plugin needs `artifactDirectory` to be set as well.
   [string]
 - `excludes` Directories to ignore under `src`. [array] [default:
-  ["**/node_modules/**", "**/__mocks__/**", "**/__generated__/**"]]
+  ["\*\*/node_modules/\*\*", "\*\*/__mocks__/\*\*", "\*\*/__generated__/\*\*"]]
```

Comment:
`**/node_modules/**` should be correct.

https://github.com/facebook/relay/blob/10e20cc7344cb9130308fb5e5b3bb6f5e95e3514/compiler/crates/relay-compiler/src/config.rs#L593-L599